src/main/java/be/polyscripts/contactmanagerapp/model/Contact.java

```diff
@@ -0,0 +1,47 @@
+package be.polyscripts.contactmanagerapp.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Builder
+@AllArgsConstructor @NoArgsConstructor @Getter @Setter
+@Entity
+@Table(name = "contact")
+public class Contact {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    @Column(name = "first_name", nullable = false)
+    private String firstName;
+
+    @Column(name = "last_name", nullable = false)
+    private String lastName;
+
+    @Column(name = "address", nullable = false)
+    private String address;
+
+    @Column(name = "tva")
+    private String tva; // example: TVA BE 0123 456 789
+
+    @Column(name = "type")

```

Comment:
keep it simple as possible, a contact that have a TVA is a freelancer, if he don't have so he is an employee :)
but your solution still working, so it's fine